lib/xandra.ex

```diff
@@ -218,12 +218,171 @@ defmodule Xandra do
   @default_address '127.0.0.1'
   @default_port 9042
 
-  @default_start_options [
-    idle_interval: 30_000,
-    default_consistency: :one,
-    protocol_version: :v3
+  # Raw NimbleOptions schema before parsing. Broken out to work around
+  # `mix format`.
+  @raw_schema = [
```

Comment:
Not sure if this is working, but it should be without the `=`:

```suggestion
  @raw_schema [
```